documentation/modules/managing/con-broker-config-properties.adoc

```diff
@@ -350,6 +342,21 @@ log.cleaner.backoff.ms=15000
 # ...
 ----
 
+.Using combined compact and delete policies 
+
+If you choose only a compact policy, your log can still become arbitrarily large. 
+In such cases, you can set the cleanup policy for a topic to compact and delete logs.
+Kafka first applies log compaction, removing older versions of records and retaining only the latest version of each key. 
```

Comment:
It's not really applying compaction "first". Both cleanup policies are applied in parallel. In practice and with default settings, compaction happens more regularly than deletion so it appears to happen "first" but you could have configs where deletion appears to happen "first".